platform/android/AndroidManifest.xml.template

```diff
@@ -1,5 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
+      xmlns:tools="http://schemas.android.com/tools"
 	  package="com.godot.game"
```

Comment:
There seems to be a mixed use of tabs and spaces in this file, could you also fix that?